### PR TITLE
fix(dmn-edit-menu): handle undo and redo entries without default

### LINKED ISSUE #1843: Keyboard shortcuts and menu entries for undo don't work in decision table
__Describe the Bug__

When I create input/output column or new rule, I cannot undo the action via menu or keyboard shortcuts.

The toolbar button still works, though.

__Steps to Reproduce__

1. Create new input in decision table
2. Press Cmd/Strg + Z or click menu entry for Undo
3. now this happens

__Expected Behavior__

Shorcut should work.

__Environment__

- OS: MacOS 10.15.4
- Camunda Modeler Version: 3.7.1, 4.0.0, develop

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

closes #1843

__Acceptance Criteria__

* [X] Corresponds to the concept: see #1843
* [X] Corresponds to the design: applied existing design which is also used [here](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/dmn/getDmnEditMenu.js#L46)

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

__Additional context__

Discovered during analysis of this PR and probably caused in same module: #1912

### CODE REVIEW

File: client/src/app/tabs/dmn/getDmnEditMenu.js [hunk 1]
```diff
@@ -61,21 +60,14 @@ export function getDmnDrdEditMenu(state) {
 }
 
 export function getDmnDecisionTableEditMenu(state) {
-  const {
-    defaultCopyCutPaste,
-    defaultUndoRedo
-  } = state;
-
-  const undoRedoEntries = defaultUndoRedo
-    ? getDefaultUndoRedoEntries()
```
Comment (MaxTru) [inline client/src/app/tabs/dmn/getDmnEditMenu.js:70]: Why would we need this?

Comment (MaxTru) [reply client/src/app/tabs/dmn/getDmnEditMenu.js:70]: I tested whether there is a difference when having this when undo/redoing in direct editing mode, but I couldn't observe one. Also `state.defaultUndoRedo` will always be `true` (see [dmnEditor.js](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/dmn/DmnEditor.js#L364)). Therefore the `getDefaultUndoRedoEntries()` will run into the [Chromium bug ](https://github.com/electron/electron/issues/3682) IMO.

Comment (MaxTru) [reply client/src/app/tabs/dmn/getDmnEditMenu.js:70]: I believe this code change is needed here: https://github.com/camunda/camunda-modeler/issues/1912

**Review by philippfromme — APPROVED** [review]
👏

**Pull request merged**